server/api/db/sqldb/db.py

```diff
@@ -1380,16 +1384,24 @@ def _find_artifacts(
         # then join the outer query on the subquery to select the correct columns of the table.
         subquery = query.subquery()
         outer_query = session.query(ArtifactV2, subquery.c.name)
+        if with_entities:
+            outer_query = outer_query.with_entities(*with_entities, subquery.c.name)
```

Comment:
How does this behave with the entities of the original query: https://github.com/mlrun/mlrun/pull/5922/files#diff-b54d4633dcd22d05679a4270bb86619cc560d09e4b68451f01d66969495a58acR1333-R1336

Reply:
wdym? this is on the outer query. you have the full artifact by then

Reply:
I mean what is returned when the query is run? will we always have the artifact & tag?

Reply:
the 1st entities filter is for the inner query which returns the art id and the tag name, then we join with the artifacts to get the full ones and if supplied the 2nd with_entitites it filters on the full artifacts and tag name